gpflow/training/tensorflow_optimizer.py

```diff
@@ -112,7 +114,9 @@ def minimize(self, model, session=None, var_list=None, feed_dict=None,
         session = model.enquire_session(session)
         with session.as_default():
             for _ in range(maxiter):
-                opt()
+                result = opt()
+                if step_callback is not None:
+                    step_callback(result)
```

Comment:
Same here, is `result` useful for step_callback?

Reply:
The reason why I'm asking is that should you pass any other information or not. @vdutor, @st-- , what do you think?